src/services/dashboards/all-dashboards/AllDashboardsPage.vue

```diff
@@ -170,6 +172,39 @@ export default {
             });
         };
 
+        const combinedDashbaordLabelsAutoCompleteHandler = (): ValueHandler | undefined => {
+            const projectLabelsValueHandler = makeDistinctValueHandler('dashboard.ProjectDashboard', 'labels');
+            const domainLabelsValueHandler = makeDistinctValueHandler('dashboard.DomainDashboard', 'labels');
+            if (!projectLabelsValueHandler && !domainLabelsValueHandler) return undefined;
+
+            return async (inputText: string, keyItem: KeyItem, currentDataType?: KeyDataType, subPath?: string) => {
+                const defaultResult = {
+                    results: [],
+                    totalCount: 0,
+                };
+                const projectResult = projectLabelsValueHandler ? await projectLabelsValueHandler(inputText, keyItem, currentDataType, subPath) : defaultResult;
+                const domainResult = domainLabelsValueHandler ? await domainLabelsValueHandler(inputText, keyItem, currentDataType, subPath) : defaultResult;
```

Comment:
suggestion: I suggest to use promise.allSettled() to await promise in parallel.

```suggestion
const promises = [];
if (projectLabelsValueHandler instanceof Promise) promises.push(projectLabelsValueHandler)
else ...
if (domainLabelsValueHandler instanceof Promise) promises.push(domainLabelsValueHandler)
else ...
const results = await Promise.allSettled(promises);
results.map(result => { ... })
```